testdriver/datasets.py

```diff
@@ -168,13 +168,31 @@ class CppVersion(Enum):
 class ICU4XVersion(Enum):
   ICU4XV1 = "1.0"
 
+# Versions for known executors
+# TODO: combine the version info
+IcuVersionToExecutorMap = {
```

Comment:
praise: I like the consolidation of data/state into fewer places

Reply:
thx